### DISCUSSION TITLE
Mechanism to start specific containers

### DISCUSSION THREAD

**Discussion by torbenj** [acorn-io/runtime#1379] [Ideas]
I have an idea to extend the run command of the acorn cli.

When working on projects that consist of multiple parts (like a backend service, a frontend service and a database) I usually use a docker-compose file to manage them.
During development I run certain services of them in the background while working on specific services in my IDE. For example when I work on the backend service I might use the docker-compose file to spin up an instance of the database service because the backend needs the database and its easier to do it that way instead of starting an instance directly via docker cli or some other mechanism.

The advantage of docker-compose in this case is that I'm able to control single services via the cli. This way I can start the database without having to start the frontend as well.

Unfortunately this isn't possible in acorn (yet). I asked on slack about this topic and I was told that I could use if conditions in combination with dev mode to just run certain services in isolation.
In my opinion this would only be a partial solution because what if I have 10 containers in my Acornfile and depending on the service I'm actively working on I need to spin up a totally different set of services? With the suggested solution I would have to constantly edit the Acornfile to accomplish this which would lead to issue down the line when I want to commit some changes to git as I would have to exclude all or some of the Acornfile changes.

I would be nice to see something like `acorn run --container database .` working that only spins up the database container (with all it's dependencies). As an optional feature this could be pluralized to `acorn run --containers database,frontend .` to spin up multiple containers at once.

What do you think about this?

**Comment by cjellick** [comment]
Hi @TorbenJ! I like this idea and definitely see the value. I'm going to open an issue from this discussion
